src/middlewares/newUser.js

```diff
@@ -0,0 +1,46 @@
+import userValidation from '../helpers.js/userValidation';
+
+class userValidate {
+  static signUp(req, res, next) {
+    const result = userValidation.SignUpschema.validate({
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      email: req.body.email,
+      password: req.body.password,
+      gender: req.body.gender,
+      passportNumber: req.body.passportNumber,
+    });
+
+    if (!result.error) {
+      req.user = result;
+      next();
+    } else {
+      if (`${result.error.details[0].path[0]}` === 'firstName') {
+        return res.status(422).json({
+          status: 422,
+          error: 'invalid firstName',
```

Comment:
This message is vague. What are the guidelines for the `firstName` field? (applies to `lastName`, `passportNumber`.

Reply:
@MuhanguziDavid  Here are the changes in consideration of your feedback

        wrong first name error message

<img width="1163" alt="Screen Shot 2020-02-13 at 10 51 53" src="https://user-images.githubusercontent.com/48021034/74417439-5db8da80-4e4f-11ea-8c15-ef4272371a9a.png">

       wrong passport number error message

<img width="1169" alt="Screen Shot 2020-02-13 at 10 53 12" src="https://user-images.githubusercontent.com/48021034/74417448-614c6180-4e4f-11ea-9350-2d0bccbcd7ea.png">

Reply:
Nice stuff